.github/workflows/build-release.yml

```diff
@@ -104,6 +104,15 @@ jobs:
           BUILD_TAG=${BUILD_TAG//[-._]/}
           echo "BUILD_TAG=$BUILD_TAG" >> $GITHUB_ENV
 
+      - name: "Update Homebrew formula"
+        uses: dawidd6/action-homebrew-bump-formula@v3
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          tap: flow-php/flow
+          formula: flow
+          tag: ${{ github.event.release.tag_name }}
+          revision: ${{ github.event.release.target_commitish }}
```

Comment:
Could you explain to me what this action is supposed to do?

Reply:
It should create PR with an update to the formula after release. But I will extract this into a new PR.